src/analysis/classic/priest/holy/CONFIG.tsx

```diff
@@ -0,0 +1,61 @@
+import { emallson } from 'CONTRIBUTORS';
+import GameBranch from 'game/GameBranch';
+import SPECS from 'game/SPECS';
+import type Config from 'parser/Config';
+import CHANGELOG from './CHANGELOG';
+
+const config: Config = {
+  // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
+  contributors: [emallson],
+  branch: GameBranch.Classic,
+  // The WoW client patch this spec was last updated.
+  patchCompatibility: '4.4.0',
+  // Update to false when the spec is mostly complete (and safe to use)
+  isPartial: true,
+  // Explain the status of this spec's analysis here. Try to mention how complete it is, and perhaps show links to places users can learn more.
+  // If this spec's analysis does not show a complete picture please mention this in the `<Warning>` component.
+  description: (
+    <>
+      Welcome! Thanks for checking out WoWAnalyzer. This guide is seeking a maintainer.
+      <br />
+      See the public GitHub repo or join our community Discord for information about contributing.
+      Thanks!
+    </>
+  ),
+  pages: {
```

Comment:
The pages section should be removed since we are no longer doing checklists. Any relevant info can be moved to the description section.